mappings/net/minecraft/block/Block.mapping

```diff
@@ -120,89 +136,152 @@ CLASS net/minecraft/class_17 net/minecraft/block/Block
-	METHOD method_1560 (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;)V
+	METHOD method_1560 onSteppedOn (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 entity
-	METHOD method_1562 (Lnet/minecraft/class_18;IIILnet/minecraft/class_25;Ljava/util/ArrayList;)V
+	METHOD method_1561 canPlaceAtConsideringMeta (Lnet/minecraft/class_18;IIII)Z
+		ARG 1 world
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 meta
+	METHOD method_1562 addBoundingBoxIfCollidingPos (Lnet/minecraft/class_18;IIILnet/minecraft/class_25;Ljava/util/ArrayList;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 box
-	METHOD method_1563 (Lnet/minecraft/class_18;IIIII)V
+		ARG 6 boundingBoxes
+	METHOD method_1563 onBlockAction (Lnet/minecraft/class_18;IIIII)V
 		ARG 1 world
-	METHOD method_1564 (Lnet/minecraft/class_18;IIILnet/minecraft/class_26;Lnet/minecraft/class_26;)Lnet/minecraft/class_27;
-		ARG 1 word
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1566 (Lnet/minecraft/class_18;III)Z
+		ARG 5 data1
+		ARG 6 data2
+	METHOD method_1564 calculateRayIntersection (Lnet/minecraft/class_18;IIILnet/minecraft/class_26;Lnet/minecraft/class_26;)Lnet/minecraft/class_27;
 		ARG 1 world
-	METHOD method_1567 (Lnet/minecraft/class_18;III)Z
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 startPos
+		ARG 6 endPos
+	METHOD method_1565 getTickrate ()I
+		COMMENT
+	METHOD method_1566 canGrow (Lnet/minecraft/class_18;III)Z
+		ARG 1 world
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_1567 canPlaceAt (Lnet/minecraft/class_18;III)Z
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1568 (Lnet/minecraft/class_14;IIII)Z
+	METHOD method_1568 isEmittingRedstonePower (Lnet/minecraft/class_14;IIII)Z
 		ARG 1 blockView
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1570 (Lnet/minecraft/class_18;IIII)Z
+		ARG 5 meta
+	METHOD method_1569 canEmitRedstonePower ()Z
+	METHOD method_1570 canTransferPowerInDirection (Lnet/minecraft/class_18;IIII)Z
 		ARG 1 world
-	METHOD method_1572 (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;Lnet/minecraft/class_26;)V
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 direction
+	METHOD method_1571 hasCollision (IZ)Z
+		ARG 1 meta
+	METHOD method_1572 onEntityCollision (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;Lnet/minecraft/class_26;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 entity
-	METHOD method_1573 (Lnet/minecraft/class_14;IIII)Z
+	METHOD method_1573 isSolid (Lnet/minecraft/class_14;IIII)Z
 		ARG 1 blockView
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
+	METHOD method_1574 getPistonPushMode ()I
+	METHOD method_1575 getBlastResisence (Lnet/minecraft/class_57;)F
+	METHOD method_1576 hasCollision ()Z
+	METHOD method_1577 setLightValue (F)Lnet/minecraft/class_17;
+		ARG 1 fractionalValue
+	METHOD method_1578 setBoundingBox (FFFFFF)V
+		ARG 1 minX
+		ARG 2 minY
+		ARG 3 minZ
+		ARG 4 maxX
+		ARG 5 maxY
+		ARG 6 maxZ
+	METHOD method_1579 isInsideYZ (Lnet/minecraft/class_26;)Z
```

Comment:
There's a similar method in `Box` which was already mapped (including the other two `isInside*`)